pandas/core/window/rolling.py

```diff
@@ -334,6 +334,13 @@ def _get_window_indexer(self) -> BaseIndexer:
         """
         if isinstance(self.window, BaseIndexer):
             return self.window
+
+        if self.is_freq_type:
```

Comment:
This clause shouldn't be needed either. `self,center` should just be passed to  `VariableWindowIndexer` in the clause below

Reply:
this is corrected too